### ISSUE TITLE
Error: Member not found: 'StorageException.fromByteArrayStream'.

### ISSUE THREAD

**Issue by Cftn** [cyber-geiger/toolbox-storage#42]
Thank you so much your hard works. 

I got an error 'Error: Member not found: 'StorageException.fromByteArrayStream'.

<img width="1348" alt="image" src="https://user-images.githubusercontent.com/15152117/155199609-d6ac369c-ec23-4183-9688-aedff1fbcb24.png">

**Comment by Cftn** [comment]
In addition, Would you please check to initialize data side? 
The local storage is empty. There is no data.

**Comment by nosadaniel** [comment]
a quick(dirty) fix for me was to remove the fromByteArrayStream method.

on testing to get the preloaded data(Global data) from the localstorage, I got a storageException Node does not exist.

**Comment by martingwerder** [comment]
This should be fixed as of 0.7.1. can you confirm

**Comment by luongnv89** [comment]
@nosadaniel @Cftn Can you confirm please?

**Comment by nosadaniel** [comment]
Thanks, @martingwerder for the quick update.
unfortunately with geigerAPI 0.7.1 new errors have been introduced and I was not enabled to compile my code.



<img width="1039" alt="image" src="https://user-images.githubusercontent.com/33612084/155312533-f2d453c3-7664-424b-a5c7-bb2c080d3a96.png">

My testing environmental
flutter version 2.10.1

**Comment by luongnv89** [comment]
@nosadaniel have you tried to uninstall the app, then tested again?
This error seems to be that you are still using the oder version of the local storage that has not introduced `castTest` function yet.

**Comment by martingwerder** [comment]
I agree with @luongnv89. This looks loke you use an outdated dependency

**Comment by Cftn** [comment]
There is no error but there is no data in local storage. 

@martingwerder, would you please check the creation of the data side that I shared you of basic data?  It exist data when I tested on Indicator using flutter testing but, there is no data when I tested on integration with UI, Flutter.

Thank you so much

**Comment by nosadaniel** [comment]
@luongnv89 and @martingwerder thanks for the info>>>
I was only using the GeigerAPi as a dependency, thinking that the geigerAPi uses the latest version of the localStorage.

FYI: GeigerApi 0.7.1 still uses geiger_localstorage: ^0.6.26.

Thanks again for the quick fix.

**Comment by martingwerder** [comment]
that is correct I will release a newer version enforcing a more resent localstorage.

**Comment by nosadaniel** [comment]
@Cftn @martingwerder concerning not ger

> There is no error but there is no data in local storage.
> 
> @martingwerder, would you please check the creation of the data side that I shared you of basic data? It exist data when I tested on Indicator using flutter testing but, there is no data when I tested on integration with UI, Flutter.
> 
> Thank you so much

@Cftn @martingwerder 
concerning not getting preloaded data.
it seems that the preloaded data from the localstorage has a visibility type red.

<img width="399" alt="image" src="https://user-images.githubusercontent.com/33612084/155347529-81083f78-5e87-4e21-99c0-a5ec63980c4d.png">

**Comment by martingwerder** [comment]
I can fix that. @Cftn: All TLP:WHITE?

**Comment by martingwerder** [comment]
Now out as 0.6.37

**Comment by Cftn** [comment]
Thank you so much Martin. 
@martingwerder, there is no data yet excepting UI data. I am not sure about which color of TLP because there is no data. 

<img width="1575" alt="image" src="https://user-images.githubusercontent.com/15152117/155351463-90bb2ec4-eaa1-4304-b756-620407d03588.png">
